### ISSUE TITLE
Send pulses with filters via the API

### ISSUE THREAD

**Issue by drorm** [metabase/metabase#5772]
Use case: we need to send customized pulses to different customers so that that we only show them information that is relevant specifically to their products.

Depends on  #5770 and  #5771 being done first.

Create an end point similar to the following:
`POST /api/pulse/:id/send`

*  **`id`** 

*  **`params`** value must be an array. The array can be empty. Filter parameters to the pulse as described in #5771

**Comment by nickcarenza** [comment]
I'd like to see pulses and alerts in general accept parameters. Should I create a separate issue for that?

**Comment by Crusader4Christ** [comment]
At least the ability to send pulse on API request.
